mllib/src/main/scala/org/apache/spark/mllib/fpm/AssociationRules.scala

```diff
@@ -142,5 +142,13 @@ object AssociationRules {
     def javaConsequent: java.util.List[Item] = {
       consequent.toList.asJava
     }
+
+    /**
+     * @return [antecedent] => [consequent]: [confidence]
```

Comment:
Sure, I don't feel strongly about it. Wouldn't this argument go for all return docs though? They're hidden by default for brevity. Maybe important ones should be put inline in the main doc but `toString` seems pretty minor.